src/metabase/api/collection.clj

```diff
@@ -103,27 +103,36 @@
   `?archived=true`.
 
   By default, admin users will see all collections. To hide other user's collections pass in
-  `?exclude-other-user-collections=true`."
-  [archived exclude-other-user-collections namespace]
+  `?exclude-other-user-collections=true`.
+
+  If personal-only is true, then return only peronsal collections."
```

Comment:
This stuff gets markdown formatted in the API docs and elsewhere

```suggestion
  If `personal-only` is `true`, then return only personal collections."
```

Reply:
Also if it actually was intentional that setting this parameter means all other parameters get ignored, you should definitely call that out in the endpoint docstring -- it's not obvious